docs/beyond_basics/101-164-dataladdening.rst

```diff
@@ -0,0 +1,141 @@
+        │   ├── anat
+        │   └── func
+        ├── sub-02
+        │   ├── anat
+        │   └── func
+        ├── sub-03
+        │   ├── anat
+        │   └── func
+        ├── sub-04
+        │   ├── anat
+        │   └── func
+        └── sub-05
+            ├── anat
+            └── func
+
+   Consider further that you want to transform all ``sub-*`` directories into individual datasets.
+   Here is a line that would do this for the example above::
+
+      $ for dir in study/sub-0{1,2,3,4,5}; do datalad -C $dir create --force; done
+
+Step 3: Saving dataset contents
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Any existing content in your newly created dataset(s) still needs to be saved into its dataset at this point (unless it was already under version control with Git).
+This can be done with the :command:`datalad save` command -- either "in one go" using a plain ``datalad save`` (saves all untracked files and modifications to a dataset -- by default into the dataset annex), or step-by-step by attaching paths to the ``save`` command.
+Make sure to run :command:`datalad status` frequently.
+
+.. findoutmore:: Save things to Git or to git-annex?
+
+   By default, all dataset contents are saved into :term:`git-annex`.
+   Depending on your data and use case, this may or may not be useful for all files.
+   Here are a few things to keep in mind:
+
+   - large files, in particular binary files should almost always go into :term:`git-annex`. If you have pure data dataset made up of large files, put it into the dataset annex.
+   - small files, especially if they are text files and undergo frequent modifications (e.g., code, manuscripts, notes) are best put under version control by :term:`Git`.
+   - If you plan to publish a dataset to a repository hosting site without annex support such as :term:`GitHub` or :term:`GitLab`, and do not intend to set up third party storage for annexed contents, be aware that only contents placed in Git will be available to others after cloning your repository. At the same time, be mindful of file size limits the services impose. The largest file size GitHub allows is 100MB -- a dataset with files exceeding 100MB in size in Git will be rejected by GitHub. :term:`Gin` is an alternative hosting service with annex support.
+
+   You can find guidance on how to create configurations for your dataset (which need to be in place and save prior to saving contents!) in the chapter :ref:`chapter_config`, in particular section :ref:`config2`.
+
+.. note::
+
+   Be mindful during saving if you have a directory that should hold more, yet uncreated datasets down its hierarchy, as a plain ``datalad save`` will save *all* files and directories to the dataset!
+
+If you are operating in a hierarchy of datasets, running a recursive save from the top-most dataset (``datalad save -d . -r``) will save you time: All contents are saved to their respective datasets, all subdatasets are registered to their respective superdatasets.
```

Comment:
But only immediate subdatasets. A single `save -r` will not register a dataset within a dataset as a subsubdataset of the root. Hence I suggested the `create -d^` approach.